extension/src/bracketMatch/bracketMatching.ts

```diff
@@ -0,0 +1,368 @@
+import * as vscode from 'vscode';
+
```

Comment:
is this file added by accident?

Reply:
Yes, it is. Looks like the git source control in vscode is doing the git add cross branches.